CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### New features
 
 * [#7274](https://github.com/rubocop-hq/rubocop/issues/7274): Add new `Lint/SendWithMixinArgument` cop. ([@koic][])
+* [#7272](https://github.com/rubocop-hq/rubocop/pull/7272): Show warning message if passed string to 'Enabled' key in .rubocop.yml. ([@unasuke][])
```

Comment:
Since other than `Enabled` are supported, therefore this changelog entry may be updated.

Reply:
Sure. That makes sense.